apps/home/templates/authenticated-home.html

```diff
@@ -107,7 +107,7 @@ <h2 class="ds-c-alert__heading">{% trans "You haven’t registered or created an
           <h2>Blue Button Resources</h2>
           <div class="bb-card-container-2-2 ds-u-margin-y--3">
 
-            <a class="bb-c-card default-card" href="https://bluebutton.cms.gov/blog/Delving-in-to-Blue-Button-Data.html">
+            <a class="bb-c-card default-card" href="https://github.com/CMSgov/bluebutton-site-static/blob/master/_posts/2018-07-24-Delving-in-to-Blue-Button-Data.md">
```

Comment:
Yes, I know it's goofy to link to a markdown file in the repo but, until the data dictionary is done, I'd prefer to link to the low traffic page this way.

Reply:
hmmm I wonder if we just shouldn't drop it all together

Reply:
The blog posts on the dashboard? Yup, I agree!

Reply:
I just meant remove this link all-together instead of linking to a markdown file in github...

Reply:
But, honestly, removing it all together is still not a bad idea. There is no referring traffic reported that I can see in analytics from the sandbox posts to the static site.